pkiinit/docker-compose.yml

```diff
@@ -45,6 +45,8 @@ services:
     hostname: edgex-pki-init
     networks:
       - edgex-network
+    tmpfs:
```

Comment:
Aren't tmpfs mounts only supported on Linux?  Should we consider something else to be cross-platform?

Reply:
point well taken- but for now, the secret store only supports on linux as the original planned (see the build in makefile having GOOS=linux).

Reply:
Another way maybe running docker engine on windows having Linux container I guess.